### ISSUE TITLE
Anime Style Space Exploration Using Metric Learning and Generative Adversarial Networks

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#446]
Xiang, Sitao, Li, Hao

Deep learning-based style transfer between images has recently become a
popular area of research. A common way of encoding "style" is through a feature
representation based on the Gram matrix of features extracted by some
pre-trained neural network or some other form of feature statistics. Such a
definition is based on an arbitrary human decision and may not best capture
what a style really is. In trying to gain a better understanding of "style", we
propose a metric learning-based method to explicitly encode the style of an
artwork. In particular, our definition of style captures the differences
between artists, as shown by classification performances, and such that the
style representation can be interpreted, manipulated and visualized through
style-conditioned image generation through a Generative Adversarial Network. We
employ this method to explore the style space of anime portrait illustrations.

https://arxiv.org/abs/1805.07997
